hotness/consumers.py

```diff
@@ -297,8 +351,8 @@ def handle_buildsys_scratch(self, msg):
         text2 = "%s %s" % (subtitle, url)
 
         # Followup on bugs we filed
-        if task_id in self.triggered_task_ids:
-            bugs.append((self.triggered_task_ids.pop(task_id), text1))
+        #if task_id in self.triggered_task_ids:
+        #    bugs.append((self.triggered_task_ids.pop(task_id), text1))
```

Comment:
Can the rebase helper logic be split into two parts?  One part that prepares the scratch build, submits it, and returns the task_id, and then another part that downloads the results and analyzes them?